### DISCUSSION TITLE
Issue with Viewing Top Committers Ranking on GitHub Account ❓

### DISCUSSION THREAD

**Discussion by mgunawardhana** [community/community#121221] [Other Feature Feedback, Questions, & Ideas]
### 🚧 Issue

🚧 Question 🚧

🚩 I am reaching out to address an issue I have encountered with my GitHub account. Recently, I have noticed that I am unable to view the top committers ranking on my account. This has caused some inconvenience as I rely on this feature to gauge my activity and contributions within the GitHub community.

Could you please assist me in resolving this matter? Specifically, I would like to know the details of my rank among the top committers, as well as any relevant statistics or metrics associated with my account's activity.

![Screenshot (240)](https://github.com/community/community/assets/100486080/f10b0b1c-f534-4a5a-9a1d-b7ba47909200)

@ashkulz , @github could you review this please ?

**Comment by dane** [comment]
This looks like a question for @ashkulz, the maintainer of committers.top. Alternatively, you could check out  [the repo](https://github.com/ashkulz/committers.top) and see what API calls are made and how the badges are calculated.

**Reply by ashkulz** [reply]
@mGunawardhana your location is "Earth", which doesn't fit into any specific country. It [might fit into Worldwide](https://committers.top/worldwide) but that requires you to have 1800+ followers.

See [the FAQ](https://github.com/ashkulz/committers.top?tab=readme-ov-file#why-am-i-not-on-this-list) for details.

**Reply by dane** [reply]
@mGunawardhana looks like you have your answer.

**Reply by mgunawardhana** [reply]
@ashkulz no my other friends have it. simply saying they don't have 1800 followers

**Reply by dane** [reply]
Would you provide an example?

**Reply by mgunawardhana** [reply]
![Screenshot (265)](https://github.com/community/community/assets/100486080/98db2a16-0351-493a-bd04-48f450637f11)

**Reply by dane** [reply]
We need a username.

**Reply by ashkulz** [reply]
@mGunawardhana I think you've changed your location recently, wait for a day or two till the data for Sri Lanka is refreshed -- you can [watch the commits](https://github.com/ashkulz/committers.top/commits/gh-pages).

**Reply by mgunawardhana** [reply]
ok @ashkulz

**Comment by midas-pro** [comment]
good
